yongkeun/week2/과제/style.css

```
@@ -0,0 +1,31 @@
+body {
+    height: 100vh;
+    background-color:coral;
+    display:flex;
+    justify-content: center;
+    align-items: center;
+}
```

```suggestion
body {
    height: 100vh;
    background-color:coral;
    display:flex;
    justify-content: center;
    align-items: center;
    margin: 0;
}
```
현재 제출하신 코드를 실행하면 스크롤이 생깁니다. 이것은 body의 margin이 기본적으로 지정되어 있기 때문인데요. 그렇기 때문에 이를 리셋해주는 코드를 body에 넣어줘야 해요. body에 margin: 0; 을 넣으면 스크롤이 사라집니다~